packages/runner-shared/src/logger.js

```diff
@@ -54,6 +55,26 @@ export const logger = {
     return Math.max(...lengths)
   },
 
+  _logArgs (consoleProps) {
+    const args = this._getArgs(consoleProps)
+
+    if (!args) return
+
+    this.log(`%cArgs:`, 'font-weight: bold')
+
+    args.forEach((arg, index) => {
+      this.log(`%c  [${index}]:`, 'font-weight: bold', arg)
+    })
```

Comment:
What do you think about keeping the old behavior if `args.length === 1`, and then only showing the indices if there is more than one element?

Reply:
I'll change it that way.

Reply:
After thinking it again, I guess not keeping the old behavior is better. The purpose of this change is to improve the readability of long strings. When we keep the old behavior and the only argument is a long string, it prints like below and it's still hard to read.

```
Args: ["long\n\n\nstring\n\nis here"]
```

With the change in this PR, it's like below:

```
Args:
   [0]: long

string

is here
```

I think the latter is what the user wants and it's easier to read.